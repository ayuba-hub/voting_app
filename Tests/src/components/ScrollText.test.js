import React from 'react';
import {shallow} from 'enzyme';
import ScrollText from './ScrollText';
describe("test on ScrollText component", () => {
    it('expects to render ScrollText component', () => {
        expect(shallow(<ScrollText/>).length).toEqual(1)
    })
    it('expects to render ScrollTextLand component', () => {
        expect(shallow(<ScrollText/>)).toMatchSnapshot();
    })
})