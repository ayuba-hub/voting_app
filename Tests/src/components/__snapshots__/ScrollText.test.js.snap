// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test on ScrollText component expects to render ScrollTextLand component 1`] = `ShallowWrapper {}`;
